sdk/src/main/scala/io/horizen/account/state/MessageProcessorUtil.scala

```diff
@@ -33,6 +33,7 @@ object MessageProcessorUtil {
       Seq(EoaMessageProcessor,
         WithdrawalMsgProcessor,
         ForgerStakeMsgProcessor(params),
+        ForgerStakeV2MsgProcessor(params),
```

Comment:
ForgerStakeV2MsgProcessor(params) should be moved before EoaMessageProcessor, as for comment line 26-29